ProjectSettings/EditorBuildSettings.asset

```diff
@@ -29,6 +29,9 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/GothicVR/Scenes/Worlds/orctempel.zen.unity
     guid: a0523f0754a2426488649326f0f27a7f
+  - enabled: 1
+    path: Assets/GothicVR/Scenes/MainMenu.unity
```

Comment:
Please move main menu scene near top. After bootstrap and general scene. Minor detail but easier for others to see when entering build settings.